### DISCUSSION TITLE
Remove automatic checklists

### DISCUSSION THREAD

**Discussion by Simleybob** [magknight/787aviatorsTracker#557] [Completed things] [answered]
When using the checklists and checking them off, by clicking on each one, I have noticed that by clicking on each item it completes that item in the aircraft. For example when checking off 'set take-off flaps' it will go to flap 5. This is not realistic, and can become very annoying if you are trying to complete the checklists when not in that stage of flight. For example when you have just taken off and want to complete the before take-off checklist, for example. Please just make it so that when you click on that item to complete it, it doesn't actually complete the task for you, it just checks it off. Thanks.

**Comment by aeroniemi** [comment]
That is literally the point of auto checklists - in reality, if you set the flaps to 5 it'd be marked as green, so it's not really a loss of realism to do it automatically. Could consider having a option to disable the autocomplete, although that'd make some things impossible to action, such as the announcements

**Reply by Simleybob** [reply]
Okay, auto checklists are great, but not vice versa - it does just get annoying when you are trying to complete the checklists, and you find that the flaps have been deployed when you've just retracted them.

**Reply by aeroniemi** [reply]
in reality, if the item said you needed flaps 5 and the flaps were not in position 5 you'd need to override the item - you can't just mark it as green if the associated function is not completed

**Reply by Simleybob** [reply]
Okay thanks

**Comment by aeroniemi** [comment, marked as answer]
They were changed slightly for 1.7.0 - issue resolved
